resources/views/home/changelog.blade.php

```diff
@@ -19,10 +19,10 @@
 
 @section('content')
```

Comment:
this file can be simplified by splitting between show and index. Also need to be relocated according to controller name and action.